src/components/UserDashboard/SavedQueries/CohortTab.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { TRACKING_EVENTS, trackUserInteraction } from 'services/analyticsTracking';
+import { Api } from 'store/apiTypes';
+import { QueryType, SavedQueriesIdToStatus, SplitSavedQueries } from 'store/SavedQueriesTypes';
+import { LoggedInUser } from 'store/userTypes';
+import { VirtualStudyPlusId } from 'store/virtualStudiesTypes';
+import { Link } from 'uikit/Core';
+
+import SavedQueriesTab from './SavedQueriesTab';
+
+type Props = {
+  loggedInUser: LoggedInUser;
+  api: Api;
+  deleteParticularSavedQuery: (
+    api: Api,
+    queryId: string,
+    loggedInUser: LoggedInUser,
+    queryType: QueryType,
+  ) => void;
+  queries: SplitSavedQueries;
+  queryIdToStatus: SavedQueriesIdToStatus;
+};
+
+const CohortTab = (props: Props) => {
+  const { queries, queryIdToStatus, loggedInUser, deleteParticularSavedQuery, api } = props;
+
+  return (
+    <SavedQueriesTab
+      items={queries.cohort}
+      queryIdToStatus={queryIdToStatus}
+      makeDescription={(item) => {
+        const vs = item as VirtualStudyPlusId;
+        return (
+          <>
+            {vs.description.length >= 140 ? `${vs.description.slice(0, 140)}...` : vs.description}
```

Comment:
a detail but i would put 140 in a variable to avoid magic number :)

Reply:
yes, good point.